src/lib/actions/projects.ts

```diff
@@ -29,15 +31,26 @@ export const getProjects = async (farcasterId: string) => {
 export const createNewProject = async (details: CreateProjectParams) => {
   const session = await auth()
 
-  if (!session?.user?.id) {
+  if (!session?.user?.id || !session.user.farcasterId) {
     return {
       error: "Unauthorized",
     }
   }
 
+  // Create project attestation
+  const attestationId = await createProjectAttestation({
+    farcasterId: session.user.farcasterId,
+  })
+
+  const projectId = nanoid()
```

Comment:
i feel like the attestation schema should also include the project ID which is why I generated it here so we can update once the schema changes